android/app/src/main/assets/config.json

```diff
@@ -1,5 +1,5 @@
 {
   "Email": "dev@fossasia.org",
   "App_Name": "Open Event",
-  "Api_Link": "https://eventyay.com/api/v1/events/6/"
+  "Api_Link": "https://api.eventyay.com/v1/events/69/"
```

Comment:
@Shailesh351 this could be done in a separate issue if'd like since it's kinda out of the scope of this PR. (Be sure to create one ... I'll assign it to you)